snuba/web/rpc/v1/resolvers/R_eap_spans/common/common.py

```diff
@@ -77,6 +77,9 @@
 
 
 def use_eap_items_table(request_meta: RequestMeta) -> bool:
+    if request_meta.referrer.startswith("force_use_eap_items_table"):
```

Comment:
I want to force using the OLD table

think about this: I run a timeseries query for some ingested timeframe, if `use_eap_items_table` is turned on, there's no way to for me to compare that query against the old table.